src/Workload/Microsoft.NET.Sdk.Maui/Microsoft.NET.Sdk.Maui.csproj

```diff
@@ -63,12 +63,19 @@
         OldValue="@VERSION@"
         NewValue="$(PackageReferenceVersion)"
     />
+    <ReplaceText
+        Input="$(IntermediateOutputPath)WorkloadManifest.json"
+        Output="$(IntermediateOutputPath)WorkloadManifest.json"
+        OldValue="@MAUI_GRAPHICS_VERSION@"
+        NewValue="$(MicrosoftMauiGraphicsVersion)"
+    />
     <ReplaceText
         Input="%(_JsonVariableMatrix.OutputPath)"
         Output="%(_JsonVariableMatrix.OutputPath)"
         OldValue="%(_JsonVariableMatrix.OldValue)"
         NewValue="%(_JsonVariableMatrix.NewValue)"
     />
+    <Message Text="##vso[task.setvariable variable=MicrosoftMauiGraphicsVersion;isOutput=true]$(MicrosoftMauiGraphicsVersion)" Importance="high" />
```

Comment:
I don't want to hold this up, b/c we probably need this in ASAP.

It looks like it was tricky to pass this value from MSBuild to .yaml. Could we have made Maui's build output the Maui.Graphics `.nupkg` files to something like `artifacts\signed`? and as long as they got copied to `nugets\signed`, that might have been a little simpler?

Reply:
why can we read it from Version.props ?